### ISSUE TITLE
Windows 10 - Fresh 1.1.1 install displays update prompt and IDs itself as 1.0.0

### ISSUE THREAD

**Issue by networked47** [GitSquared/edex-ui#312]
After both updating and doing a fresh install of the 1.1.1 package on Windows 10 the prompt to update to 1.1.1 still presents itself on each startup. 

On the 'boot screen' eDEX Kernel Version is 1.0.0. Should this not reflect the current version (or maybe it is cosmetic)

**Comment by GitSquared** [comment]
The boot screen version is cosmetic. I'll test the package tonight, this is a weird bug.

Thanks for the report.

**Comment by networked47** [comment]
I see. :) 

Let me know if you need any more information from me.

**Comment by networked47** [comment]
I did a check with another computer I have (different OS (macOS) but I had an inkling)... the version that the Windows version installs is either not 1.1.1 or it fails to update the files for some reason. Windows defender goes in and I acknowledge the install, but during the install process the installer tells me that edex is running and asks me press ok to close it.. which I do. The install then seems to commence.

The version that runs on my windows machine is defiantly older since things like the multitabs and satellites on the world map don't appear. This however also happens on a clean install.

**Comment by networked47** [comment]
Hi again, sorry to just dump message on message...

I think I might have found what it is.

When trying to download from this location: https://github.com/GitSquared/edex-ui/tree/v1.1.1 
The link in the Readme.md is: Download 1.0.1: for Windows | for macOS | for Linux x32 | for Linux x64 and the windows link goes to the file: https://github.com/GitSquared/edex-ui/releases/download/v1.1.0/eDEX-UI.Windows.Installer.exe

Going on another link ( https://github.com/GitSquared/edex-ui/releases/tag/v1.1.1 ) that I think I got through the UI this time I downloaded a new installer, ran it (no error that edex was running even though I was not executing it). And now the screen I get compares for the most part equal to the other install mentioned above. 

So, it was just an oversight in the Readme.md that I didn't see before but now payed way closer attention to. :)
